src/openforms/submissions/tokens.py

```diff
@@ -54,5 +59,28 @@ def get_hash_value_parts(self, submission: Submission) -> List[str]:
         return [str(getattr(submission, attr)) for attr in attributes]
 
 
+class SubmissionResumeTokenGenerator(BaseTokenGenerator):
+    key_salt = "openforms.submissions.tokens.SubmissionResumeTokenGenerator"
+
+    def get_token_timeout_days(self, submission: Submission) -> int:
+        """
+        Number of days before the submission would be removed
+        """
+        return (
+            submission.form.incomplete_submissions_removal_limit
+            or GlobalConfiguration.get_solo().incomplete_submissions_removal_limit
+        )
+
+    def get_hash_value_parts(self, submission: Submission) -> List[str]:
+        """
+        Obtain the attribute values that mutate to invalidate the token.
+        """
+        attributes = [
+            "suspended_on",
+        ]
```

Comment:
I'd take `completed_on` into consideration too, so that a completed submission cannot be altered anymore after it started processing. Please add a test for that too.